src/pages/components/checkbox/usage.mdx

```diff
@@ -1,71 +1,261 @@
+- Regardless of whether the label is visible in the interface, a group label is always needed in code, whether it's for one checkbox or a group of them. See the [checkbox code tab](https://www.carbondesignsystem.com/components/checkbox/code) for more information.
+
+#### Checkbox labels
+
+- Always use clear and concise labels for checkboxes.
+- Be explicit about the results that will follow if the checkbox is selected.
+- Labels appear to the right of checkbox controls.
+
+### Overflow content
+
+- We recommend checkbox labels being fewer than three words.
+- If you are tight on space, consider rewording the label. Do not truncate checkbox label text with an ellipsis.
+- Long labels may wrap to a second line, and this is preferable to truncation.
+- Text should wrap beneath the checkbox so the control and label are top aligned.
+
+<Row>
+<Column colLg={8}>
+
+![Text wrapping for checkboxes.](images/checkbox-usage-10.png)
+
+</Column>
+</Row>
+
+<DoDontRow>
+  <DoDont caption="Do let text wrap beneath the checkbox so the control and label are top aligned.">
+
+![Do let text wrap beneath the checkbox so the control and label are top aligned.](images/checkbox-usage-11-do.png)
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not vertically center wrapped text with the checkbox.">
+
+![Do not vertically center wrapped text with the checkbox.](images/checkbox-usage-11-don't.png)
+
+  </DoDont>
+</DoDontRow>
+
+### Further guidance
+
+For further content guidance, see Carbon's [content guidelines](notion://www.notion.so/guidelines/content/general).
+
+## Behaviors
+
+### States
+
+The checkbox input allows for a series of states: **unselected**, **selected**, and **indeterminate**. The default view of a set of checkboxes is having no option selected.
+
+<Row>
+<Column colLg={8}>
+
+![Unselected, selected and indeterminate states of checkbox.](images/checkbox-usage-7.png)
+
+</Column>
+</Row>
+
+Use the indeterminate state when the checkbox contains a sublist of selections, some of which are selected, and some unselected. In addition to unselected, selected, and indeterminate states, checkboxes also have states for focus and disabled.
+
+<Row>
+<Column colLg={8}>
+
+![Focus and disabled states of checkbox.](images/checkbox-usage-8.png)
+
+</Column>
+</Row>
+
+### Interactions
+
+#### Mouse
+
+Users can trigger an item by clicking on the checkbox input directly or by clicking on the checkbox label. Having both regions interactive creates a more accessible click target.
+
+<Row>
+<Column colLg={8}>
+
+![Click targets for checkbox.](images/checkbox-usage-9.png)
+
+</Column>
+</Row>
+
+#### Keyboard
+
+Users can trigger a state change by pressing the `Enter` or `Spacebar` key while the checkbox input has focus. For additional keyboard interactions, see the [accessibility tab](https://www.carbondesignsystem.com/components/checkbox/accessibility).
```

Comment:
is `Enter` to select/deselect expected behavior? we currently don't have this and it's not default behavior for checkboxes

Reply:
Okay i will take that out

Reply:
by default, space bar selects and deselects but Enter doesn't, and our components follow that pattern

Reply:
I didn't see your edit originally, but I don't think JAWS and NVDA change the interaction pattern (not entirely sure on that though since I don't have access to either)

Reply:
@dakahn do you have any insight on this?

Reply:
Space (not enter) to select and deselect is expected default behavior. Unless we have really really good reason to we defer to the [WAI-Aria Authoring Practices Guide](https://www.w3.org/TR/2019/NOTE-wai-aria-practices-1.1-20190814/examples/checkbox/checkbox-1/checkbox-1.html) for keyboard interaction patterns.